src/app/[locale]/api/convert/[templateId]/route.ts

```diff
@@ -23,6 +23,7 @@ export const POST = withApiAuth(async (req: NextRequest, { params }: { params: {
       );
     }
 
+    const clientId = req.headers.get('client_id');
```

Comment:
unnecessary, by doing this you are essentially changing the external APIs contract!!
Be very much mindfull while change contract of this API